src/process/process.hh

```diff
@@ -185,7 +185,7 @@ namespace SSC {
 
     int wait () {
       do {
-        std::this_thread::yield();
+        msleep(256);
```

Comment:
maybe we can use something similar to this?

https://en.cppreference.com/w/cpp/thread/yield

Reply:
There's https://en.cppreference.com/w/cpp/thread/sleep_for which looks like what we need

Reply:
That is exactly what `msleep()` does :)

https://github.com/socketsupply/socket/blob/master/src/common.hh#L970-L972

Reply:
oh great!

Reply:
@chicoxyzzy The `yield()` + loop approach might be the right approach for `msleep()`, but it's worth noting that `yield()` alone wasn't working here in terms of lower CPU usage. `msleep()` does call `yield()`, but it also calls `sleep()`.